src/libslic3r/GCode.cpp

```diff
@@ -1102,6 +1102,7 @@ void GCode::_do_export(Print &print, FILE *file)
 		if (out_filament_cost.second)
 			_writeln(file, out_filament_cost.first);
     }
+    _writeln(file, "; ERIK WAS HERE \n");
```

Comment:
huh?

Reply:
Ha!  This was left behind from when I was poking the code with a stick to see how it worked.   I'll remove it.  Thanks for noticing.